packages/dev/addons/src/lottie/worker.ts

```diff
@@ -0,0 +1,48 @@
+import type { AnimationConfiguration } from "./lottiePlayer";
+import { AnimationController } from "./rendering/animationController";
+
+let AnimationPlayer: AnimationController | null = null;
+
+const LoopAnimation = false; // By default do not loop animations

```

Comment:
I feel like this would be easier to follow if you just had a default options object and merged the passed in options with the default.

Reply:
Good idea! I'll address.